models/src/main/scala/coop/rchain/models/block/StateHash.scala

```diff
@@ -10,4 +11,12 @@ object StateHash {
   implicit class StateHashOps(bs: StateHash) {
     def base16String: String = Base16.encode(bs.toByteArray)
   }
+
+  implicit def ordering: Ordering[StateHash] =
+    Ordering.by((b: ByteString) => b.toByteArray.toIterable)
```

Comment:
Why use `toIterable` here? It is kind of reduntant here.

Reply:
I copied this ordering from Estimator, not really sure why `toIterator` is used - maybe it has some performance benefits

Reply:
`toIterator` is differenct `toIterable`. Here `toIterable` just return instance itself so we don't need that.

Reply:
just `b.toByteArray` won't work because there is not implicit ordering by `Array[Byte]`. Thats why it is converted to iterable